src/nulog-inference-service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7-slim
```

Comment:
3.7 because some python libs version is related to python version, they may not be available in 3.8. 
`slim` because it makes the docker image lighter.